external-crates/move/crates/move-analyzer/trace-adapter/src/trace_utils.ts

```diff
@@ -48,14 +53,31 @@ interface JSONTraceCompound {
     variant_tag?: number;
 }
 
-interface JSONTraceRuntimeValue {
-    value: JSONTraceValueType;
+interface JSONTraceRefValueContent {
```

Comment:
These are changes to accommodate JSON data describing references with some renaming of existing interfaces to disambiguate things